spec/models/manageiq/providers/nsxt/network_manager/refresher_spec.rb

```diff
@@ -2,10 +2,10 @@
   it ".ems_type" do
     expect(described_class.ems_type).to eq(:nsxt)
   end
-  
+
   context "#refresh" do
     let(:ems) do
-      hostname = Rails.application.secrets.nsxt.try(:[], :hostname) || "NSXTHOSTNAME"
+      hostname = Rails.application.secrets.nsxt.try(:[], :hostname) || "nsxthostname"
```

Comment:
rest-client downcases the hostname so had to update it otherwise there were vcr unhandled http interaction errors